src/components/TechGala/Projects/ProjectCard/ProjectCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Tag from './Tag/Tag';
+
+function ProjectCard(props) {
+    return (
```

Comment:
This is perhaps personal preference, but I'm a big fan of "object destructuring". I think it makes code more readable and makes things DRYer (DRY is an acronym for "Don't Repeat Yourself").

For example you can do

```suggestion
const {image, title, winner, summary, names, url, category} = props.project;
    return (
```
to create a bunch of local variables holding the information you want, then you can simply reference these variables instead of the `props.project.` object in your returned HTML.